### ISSUE TITLE
Wrong username/password

### ISSUE THREAD

**Issue by MBakac** [Schmavery/facebook-chat-api#767]
Hi, the api keeps throwing the "Wrong username/password" error. I tried with a few different accounts including a whitehat one and it didn't work.

`martin@MartinPC2:~/Documents/fbapitest$ node index.js
info login Logging in...
ERR! login Wrong username/password.
{ error: 'Wrong username/password.' }
`

**Comment by KMatuszak** [comment]
1. Does your password contain characters such as `\`?
2. Show the code you are using

**Comment by MBakac** [comment]
1. It does not, I changed it to include only letters and numbers.
2. 
![image](https://user-images.githubusercontent.com/10883711/70071735-119f1d80-15f6-11ea-86ca-25058f261f60.png)
3. I'm using "facebook-chat-api": "^1.6.1" from npm.

**Comment by KMatuszak** [comment]
1. Uninstall current version of API: `npm uninstall facebook-chat-api`
2. Install version from github repo `npm install Schmavery/facebook-chat-api`
3. Check again

**Comment by MBakac** [comment]
I uninstalled the old version and installed the github version and am still experiencing the same issue.

**Comment by KMatuszak** [comment]
Do you enter email and password between the characters `<` and `>`?

**Comment by MBakac** [comment]
Nope, I enter it like "someone@something.com" and "password". I tried using the pychat api and it was able to log in fine.

**Comment by adobebeta** [comment]
> Nope, I enter it like "[someone@something.com](mailto:someone@something.com)" and "password". I tried using the pychat api and it was able to log in fine.

Im meet same problem today, how you fix this problem? T T

**Comment by LeHungTin** [comment]
@adobebeta Hey, https://www.youtube.com/watch?v=y9_yd5a3scM

**Comment by ttoavina** [comment]
I have the same issues

**Comment by ttoavina** [comment]
> @adobebeta Hey, https://www.youtube.com/watch?v=y9_yd5a3scM

Thank you, this video saved me
